docs/index.md

```diff
@@ -343,10 +343,14 @@ you're done. For specific instructions on a number of common hosts, see the
 To get help with MkDocs, please use the [discussion group], [GitHub issues] or
 the MkDocs IRC channel `#mkdocs` on freenode.
 
+[guide]: user-guide/
```

Comment:
This results in a `404 Not Found` error as there is no index page for any subdirs in the docs. Let's just remove the link for now. This can be addressed more fully when we get to #1629.

Otherwise this looks good. Thanks.

Reply:
Fixed in the PR